app/observability.py

```diff
@@ -39,8 +45,15 @@ def get_exporter(
     return (OTLPSpanExporter(endpoint, True), OTLPMetricExporter(endpoint, True))
 
 
-def setup_otel_exporter(app_name: str, endpoint: str = "localhost:4317"):
-    logger.info("Starting opentelemetry exporter", extra={"endpoint": endpoint})
+def setup_otel_exporter(app_name: str, endpoint: str):
+    if settings.running_unittests == 1:
+        # If we're running unittests, skip setting up exporter and provider so
+        # it's using the default no-op things
+        return
```

Comment:
I managed to get tests working with and without this. However, I decided that we should be doing what they recommend, though it's recommended in a GitHub issue comment:

https://github.com/open-telemetry/opentelemetry-python/issues/3480#issuecomment-1781507977

I could be convinced to run the unittests with the otel-collector service. Next step is to figure out how to assert what traces/metrics were emitted. Maybe that'll be easier if we have the otel-collector running.